specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-04-01-preview/connectedClusters.json

```diff
@@ -149,6 +149,12 @@
               "$ref": "#/definitions/ConnectedCluster"
             }
           },
+          "202": {
+            "description": "Resource patch request accepted",
```

Comment:
do we need to add this in other api versions as well ?.

Reply:
No, we only need to add in the api-versions where patch async operations is supported.